python/tvm/relay/op/transform.py

```diff
@@ -836,7 +836,7 @@ def broadcast_to(data, shape):
         The resulting tensor.
     """
     if isinstance(shape, Constant):
-        shape = list(shape.data.numpy())
+        shape = [int(i) for i in shape.data.numpy()]
```

Comment:
It's possible, although the default handler for a python list of ints would hit the same problem. In either case, I'll push a proper fix for the reduction schedule on CUDA (forcibly cast the extent to `int32`, since that's needed for CUDA anyway if I understand correctly).

Do you think I should wrap ints using numpy `int64` to avoid this problem? I'm slightly worried it will break assumptions elsewhere.

Reply:
I guess one possible option is to change the behavior `tvm.runtime.convert` to check for overflow and use `int64` when necessary